garrysmod/lua/includes/extensions/util.lua

```diff
@@ -365,3 +365,33 @@ function util.RemovePData( steamid, name )
 	sql.Query( "DELETE FROM playerpdata WHERE infoid = " .. SQLStr( name ) )
 
 end
+
+--[[---------------------------------------------------------
+	Name: BinaryModuleInstalled( name )
+	Desc: Returns whether a binary module with the given name is present on disk
+-----------------------------------------------------------]]
+do

```

Comment:
What is this for?

Reply:
Scope encapsulation. Just separates `suffix` and `fmt` from the rest of the code, it's not necessary, just a Lua pattern

Reply:
I get that, my point is that nothing else in the codebase uses this.